Library/Formula/tesseract.rb

```diff
@@ -17,6 +17,7 @@ class Tesseract < Formula
     depends_on "automake" => :build
     depends_on "libtool" => :build
     depends_on "pkg-config" => :build
+    depends_on "cairo" => :build
```

Comment:
Cairo is unlikely to be a :build dependency; the shared libraries are probably needed once the tools are installed.